tests/functional/storageclass/test_cephfilesystem_creation.py

```
@@ -25,35 +25,35 @@ def test_Cephfilesystem_creation(self):
                 cephfs_name="test-ceph-fs", label={"use": "test"}
             )
-            logger.info(
-                f"Not able to create a new ceph fs using same name {new_cephFS_obj}"
-            )
+            logger.info(f"CephFileSystem recreation returned: {new_cephFS_obj}")
 
         except CommandFailed as e:
             if "Error from server (AlreadyExists)" in str(e):
-                logger.info("Test success!")
+                logger.info("AlreadyExists error received as expected")
                 assert "Error from server (AlreadyExists)" in str(e)
```

Ruff (0.15.15)

[warning] Found assertion on exception `e` in `except` block, use `pytest.raises()` instead

(PT017)